### PR TITLE
Add Grafana link to dashboard

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
Add Grafana link to dashboard and restructure mla related tabs

### Special notes for your reviewer
FYI: We decided to reorder the MLA stuff in the cluster details view a little bit and combine previous `User Cluster Alertmanager` & `User Cluster Recording & Alerting Rules` tabs to one new tab `Monitoring, Logging & Alerting`.

mla-tab

### Release note

```release-note
Added link to grafana to ui. Visible if user cluster monitoring or user cluster logging is enabled.
```

### CODE REVIEW

**Comment by kgroschoff** [comment]
wait for https://github.com/kubermatic/kubermatic/pull/7608
/hold

**Comment by kgroschoff** [comment]
PTAL @floreks

File: src/app/cluster/cluster-details/mla/alertmanager-config/component.ts [hunk 1]
```diff
@@ -63,25 +69,29 @@ export class AlertmanagerConfigComponent implements OnInit, OnDestroy {
...
-      'https://' +
-      this._settings.mlaAlertmanagerPrefix +
-      '.' +
-      this._seed +
-      '.' +
-      window.location.hostname +
-      '/' +
-      this.cluster.id
-    );
+    return 'https://' + prefix + '.' + this._seed + '.' + window.location.hostname + '/' + this.cluster.id;
```
Comment (floreks) [inline src/app/cluster/cluster-details/mla/alertmanager-config/component.ts:94]: Please do not use `window` directly. Inject it through the construtor.

**Pull request merged**